src/server.tsx

```diff
@@ -95,7 +97,8 @@ const buildEpic = async (
             />,
         ),
     );
-    return { html, css };
+
+    return { html, css, testName: test.name, variantName: variant.name };
```

Comment:
What if we add a `meta` object to the main object (alongside `html` and `css`) and then put these 2 properties inside?
```suggestion
    return { html, css, meta: { testName: test.name, variantName: variant.name }};
```

Reply:
Yeah sounds good, I was wondering the same thing. Mostly didn't do it because I couldn't think of a suitable name!

Reply:
`meta` makes sense here I think.